src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -33,9 +36,9 @@
  * @author Jonathan Fuerth
  * @author Ognjen Bubalo
  */
-public final class JacocoBuildAction extends CoverageObject<JacocoBuildAction> implements HealthReportingAction, StaplerProxy, Serializable {
+public final class JacocoBuildAction extends CoverageObject<JacocoBuildAction> implements HealthReportingAction, StaplerProxy, Serializable, RunAction2 {
 
-	public final AbstractBuild<?,?> owner;
+	public transient Run<?,?> owner;
```

Comment:
but a `Run` is not always an `AbstractBuild` so anyone expecting this to be an `AbstractBuild` may now blow up.
This is regardless of any binding that may or not have broken (I think the variable will actually be ok - but the type changing is not).  Why this needs to be `public` baffles me (but that is a different issue)